code/game/objects/effects/anomalies.dm

```diff
@@ -221,12 +225,12 @@
 
 /obj/effect/anomaly/flux/detonate()
 	if(explosive)
-		explosion(src, 1, 4, 16, 18) //Low devastation, but hits a lot of stuff.
+		explosion(src, 1, 4, 16, 18, FALSE) // Set adminlog to FALSE for custom logging
+		if(drops_core)
+			log_and_message_admins("A [name] has detonated at [impact_area][ADMIN_COORDJMP(impact_area)]")
```

Comment:
Could consider adding this to grav, pyro and cryo anomalies aswell

Reply:
Yeah I can look into updating it, I just wanted to override that explosion log for something a bit more informative to the flux anomaly itself.

Reply:
Updated!